deepchem/feat/bert_tokenizer.py

```diff
@@ -0,0 +1,60 @@
+from deepchem.feat import Featurizer
+from typing import List
+try:
+  from transformers import BertTokenizerFast
+except ModuleNotFoundError:
+  raise ImportError(
+      'Transformers must be installed for BertFeaturizer to be used!')
+  pass
+
+
+class BertFeaturizer(Featurizer):
+  """Bert Featurizer.
+
+  Bert Featurizer.
+  The Bert Featurizer is a wrapper class for HuggingFace's BertTokenizerFast.
+  This class intends to allow users to use the BertTokenizer API while
+  remaining inside the DeepChem ecosystem.
+
+  Examples
+  --------
+  >>> from deepchem.feat import BertFeaturizer
+  >>> from transformers import BertTokenizerFast
+  >>> tokenizer = BertTokenizerFast.from_pretrained("Rostlab/prot_bert", do_lower_case=False)
+  >>> featurizer = BertFeaturizer(tokenizer)
+  >>> feats = featurizer.featurize('D L I P [MASK] L V T')
+
+  Notes
+  -----
+  Examples are based on RostLab's ProtBert documentation.
+
+  This class may contain code and/or documentation taken from the
+  RobertaFeaturizer pull request (#2581), which have been moved here due to
```

Comment:
Can you remove this note? Remember readers on the docs don't follow the details of our development process so this will just confuse them :)

Reply:
Sounds good, will do